### DISCUSSION TITLE
How to build in a sub directory?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#244] [General]
I want to test with both Clang and GCC. Currently, I have to keep two versions of the `eepp` source tree: `eepp-gcc` and `eepp-llvm`. I'm totally lacking experience with `premake`. But with `CMake`, it's very easy. You only need one source tree. You can create different build directories (e.g., `build-gcc`, `build-clang`,...) to build with different configurations.

**Comment by SpartanJ** [comment]
It's not possible with the current build configuration. This is not something that premake provides in a direct way. Paths are basically defined in the premake files by the developer.
